src/Engine/Rendering/Data/Buffer.hpp

```
@@ -37,11 +39,38 @@ namespace Buffer {
 	};
 
 
-	// Transient global data for rendering one frame.
+	// Transient handoff data between PhysicsSystem and RenderSystem for rendering one frame
+	struct PhysRendFramePacket {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: entities, epoch, simulationTime

(cppcoreguidelines-pro-type-member-init)